pkg/commands/ignore.go

```diff
@@ -53,9 +60,12 @@ Examples:
 		newIgnoreShowCommand(),
 		newIgnoreAddCommand(),
 		newIgnoreRemoveCommand(),
+		newIgnorePullCommand(),
 		newIgnoreMigrateCommand(),
 	)
 
+	zerolog.SetGlobalLevel(zerolog.ErrorLevel)
```

Comment:
Currently `--debug` and `--log-level` flags are Scan flags, so they are not available in this action. 
Do we want to move them to General flags, so we can use them here?

Reply:
Think this would make a lot of sense

Reply:
^ https://github.com/Bearer/bearer/pull/1257